src/display.jl

```diff
@@ -23,8 +23,13 @@ The following options are available:
 - `sigfigs`: number of significant figures to show in `standard` mode
 
 - `decorations` (boolean):  show decorations or not
+
+Example:
+```
+julia> setdisplay(:full, decorations=true)
+```
 """
-function displaymode(;decorations=nothing, format=nothing, sigfigs=-1)
+function setdisplay(format=nothing; decorations=nothing, sigfigs::Integer=-1)
```

Comment:
I guess there is a reason why the default is `nothing` for each kw, but why is this better than setting them directly to the values specified in `display_params`?